service/gcs/main.go

```diff
@@ -110,6 +111,43 @@ func main() {
 		bridgeOut = bridgeCon
 	}
 
+	// Setup the UVM cgroups to protect against a workload taking all available
```

Comment:
@jstarks - Mind double checking the idea here. Thanks

Reply:
Seems like a reasonable start. Maybe add some command line parameters to configure the sizes, though, so that we can adjust this from the host.